src/main/java/com/bio4j/angulillos/TypedElementIndex.java

```diff
@@ -12,17 +12,13 @@
   P extends Property<FT,X>,
   X
 >
-{
-
-  /* Index name */
-  String name();
-
-  // /* The graph */
-  // G graph();
+extends HasLabel {
 
   /* Get the indexed property. */
   P property();
 
+  public default UntypedGraph untypedGraph() { return property().elementType().graph().raw(); }
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Avoid modifiers which are implied by the context](https://www.codacy.com/app/era7/angulillos/file/2613161932/issues/source?bid=3184361&fileBranchId=3311693#l20)